### DISCUSSION TITLE
Stabilization for camera with no gyroscope metadata

### DISCUSSION THREAD

**Discussion by ninjatech971** [latenitefilms/BRAWToolbox#80] [Feature Requests]
Hi,
I really like this tool but noticed that I don't have access to FCPX built-in stabilization.
Is it possible to enable this or create a tool that would work for non-Blackmagic camera shooting BRAW (Lumix, Sony, etc)?
I mean camera with no gyroscope metadata.
That would be great!
Thanks very much!

<img width="1111" alt="Screenshot 2023-02-11 at 10 38 37 AM" src="https://user-images.githubusercontent.com/78880536/218267175-70e62479-fb11-4bf4-93b5-e8905d9ee7c5.png">

**Comment by latenitefilms** [comment]
Apologies for the delayed reply. Unfortunately this is a limitation of Final Cut Pro - it doesn't allow you to stabilise Synchronised Clips or Multicam Clips.

I definitely recommend sending Apple Feedback [here](https://www.apple.com/feedback/finalcutpro/) to explain why this functionality would be helpful for you.

As a workaround, you can try something like [CoreMelt's Lock and Load X](https://coremelt.com/products/lock-and-load-x).

I will also continue working with the Gyroflow Team to see if it's possible to add optical only stabilisation to Gyroflow - discussed more here: https://github.com/gyroflow/gyroflow/issues/45

**Comment by ninjatech971** [comment]
Hi @latenitefilms ,
No worry, I know you are very busy as it's a well awaited tool that we all wanted :).
OK I understand better. About Gyroflow, it's really up to you if you think that other people might need this option (just to make sure you don't wasted your time for only one person requesting it ahah).
I'll have a look at the Coremelt's plugin.
Thanks again for this plugin and your answer!
